tcms/testcases/models.py

```diff
@@ -279,7 +279,7 @@ class BugSystem(TCMSActionModel):
 """,
     )
 
-    api_url = models.CharField(
+    api_url = models.CharField(# pylint:disable=form-field-help-text-used
```

Comment:
All of the comments trigger flake8 errors. Check the logs or see Makefile how to execute flake8 locally.